contracts/vaults/NFTVault.sol

```diff
@@ -104,37 +150,49 @@ contract NFTVault is AccessControlUpgradeable, ReentrancyGuardUpgradeable {
     /// @notice Total outstanding debt
     uint256 public totalDebtAmount;
     /// @dev Last time debt was accrued. See {accrue} for more info
-    uint256 public totalDebtAccruedAt;
+    uint256 private totalDebtAccruedAt;
     uint256 public totalFeeCollected;
-    uint256 internal totalDebtPortion;
+    uint256 private totalDebtPortion;
 
     VaultSettings public settings;
 
     /// @dev Keeps track of all the NFTs used as collateral for positions
     EnumerableSetUpgradeable.UintSet private positionIndexes;
 
-    mapping(uint256 => Position) private positions;
+    mapping(uint256 => Position) public positions;
     mapping(uint256 => address) public positionOwner;
-    mapping(bytes32 => uint256) public nftTypeValueETH;
```

Comment:
This change will break the DefiLlama adapter once deployed (fetching floor price for EtherRocks). Can you make sure the adapter is updated at the same time or at least lmk when you plan to upgrade and I can update the adapter.

https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/jpeg-d/index.js

Reply:
Thank you for letting me know. The upgrade is most likely gonna happen today.